pkg/client/src/app/layout/AppAboutModal/tests/__snapshots__/AppAboutModal.test.tsx.snap

```diff
@@ -113,7 +113,7 @@ exports[`AppAboutModal 1`] = `
         <TextListItem
           component="dd"
         >
-          2.0
+          2.1
```

Comment:
Can we use the constant here as well?

Reply:
I don't think that's helpful since snapshots are automatically generated.